### PR TITLE
streaming map edits

### PR DESCRIPTION
**Pull request by andrew-wm-arthur**

### CODE REVIEW

File: bats/compatibility/corona-virus-test.sh [hunk 1]
```diff
@@ -99,7 +99,7 @@ local_bin="`pwd`"/"$bin"
 PATH="$local_bin":"$PATH" dolt clone Liquidata/corona-virus
 pushd "corona-virus"
 PATH="$local_bin":"$PATH" export_tables "-pre"
-dolt migrate
+time dolt migrate
```
Comment (andrew-wm-arthur) [inline bats/compatibility/corona-virus-test.sh:102]: worth keeping this time

File: go/libraries/doltcore/table/typed/noms/noms_map_updater.go [hunk 2]
```diff
@@ -132,6 +132,41 @@ func (nmu *NomsMapUpdater) WriteRow(ctx context.Context, r row.Row) error {
 	return nil
 }
 
+// WriteRow will write a row to a table
```
Comment (bheni) [inline go/libraries/doltcore/table/typed/noms/noms_map_updater.go:135]: Fix Comment

File: go/libraries/doltcore/table/typed/noms/noms_map_updater.go [hunk 3]
```diff
@@ -96,6 +96,11 @@ func (nmu *NomsMapUpdater) GetSchema() schema.Schema {
 
 // WriteRow will write a row to a table
 func (nmu *NomsMapUpdater) WriteRow(ctx context.Context, r row.Row) error {
+	return nmu.WriteEdit(ctx, r.NomsMapKey(nmu.sch), r.NomsMapValue(nmu.sch))
+}
+
+// WriteEdit will write an edit to a table's edit accumulator
+func (nmu *NomsMapUpdater) WriteEdit(ctx context.Context, pk types.LesserValuable, fieldVals types.Valuable) error {
```
Comment (bheni) [inline go/libraries/doltcore/table/typed/noms/noms_map_updater.go:103]: It's beautiful

Comment (andrew-wm-arthur) [reply go/libraries/doltcore/table/typed/noms/noms_map_updater.go:103]: 🔥

**Pull request merged**